addons/website_sale_product_configurator/controllers/main.py

```diff
@@ -73,29 +80,35 @@ def cart_options_update_json(self, product_and_options, goto_shop=None, lang=Non
         if product_and_options:
             # The main product is the first, optional products are the rest
             main_product = product_and_options[0]
-            value = order._cart_update(
+            values = order._cart_update(
                 product_id=main_product['product_id'],
                 add_qty=main_product['quantity'],
                 product_custom_attribute_values=main_product['product_custom_attribute_values'],
                 no_variant_attribute_values=main_product['no_variant_attribute_values'],
                 **kwargs
             )
 
-            if value['line_id']:
+            lines_ids = [values['line_id']]
```

Comment:
I suggested `line_ids` but as you prefer.

Reply:
<img src="https://media3.giphy.com/media/COYGe9rZvfiaQ/giphy.gif"/>